suite-common/wallet-core/src/transactions/transactionsThunks.ts

```diff
@@ -105,20 +106,22 @@ export const replaceTransactionThunk = createThunk<ReplaceTransactionThunkParams
 );
 
 interface AddFakePendingTransactionParams {
-    transaction: Transaction;
     precomposedTx: PrecomposedTransactionFinal;
     account: Account;
 }
 
 export const addFakePendingTxThunk = createThunk<AddFakePendingTransactionParams>(
     `${TRANSACTIONS_MODULE_PREFIX}/addFakePendingTransaction`,
-    ({ transaction, precomposedTx, account }, { dispatch, getState }) => {
+    ({ precomposedTx, account }, { dispatch, getState, rejectWithValue }) => {
```

Comment:
nit: Please use full name for better readability

```suggestion
    ({ precomposedTransaction, account }, { dispatch, getState, rejectWithValue }) => {
```

Reply:
fixed 7b2265b66b